### PR TITLE
move validator pubkey into startAuction

### PR DESCRIPTION
**Pull request by 0xAshish**

### CODE REVIEW

**Comment by imyourm8** [comment]
LGTM

File: contracts/staking/stakeManager/StakeManager.sol [hunk 1]
```diff
@@ -235,10 +245,10 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
...
     }
 
-    // Housekeeping function. @todo remove later
-    function forceUnstake(uint256 validatorId) external onlyOwner {
-        _unstake(validatorId, currentEpoch);
-    }
+    // // Housekeeping function. @todo remove later
+    // function forceUnstake(uint256 validatorId) external onlyOwner {
+    //     _unstake(validatorId, currentEpoch);
+    // }
```
Comment (jdkanani) [inline contracts/staking/stakeManager/StakeManager.sol:251]: Should we keep this for some times?

File: contracts/staking/stakeManager/StakeManager.sol [hunk 2]
```diff
@@ -712,15 +722,15 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
...
-    }
+    // function reinitialize(
+    //     address _NFTContract,
+    //     address _stakingLogger,
+    //     address _validatorShareFactory
+    // ) external onlyGovernance {
+    //     NFTContract = StakingNFT(_NFTContract);
+    //     logger = StakingInfo(_stakingLogger);
+    //     factory = ValidatorShareFactory(_validatorShareFactory);
+    // }
```
Comment (jdkanani) [inline contracts/staking/stakeManager/StakeManager.sol:733]: We should keep this since we may need to change the validator share factory.

Comment (0xAshish) [reply contracts/staking/stakeManager/StakeManager.sol:733]: for validator share factory we have method to update it.

Comment (0xAshish) [reply contracts/staking/stakeManager/StakeManager.sol]: keeping this one, removed Factory update function.

**Pull request merged**